.github/workflows/python2-3-app.yml

```
@@ -0,0 +1,43 @@
+name: DB12 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: False
+      matrix:
+        python_version:
+          - 2.7
+          - 3.6
+          - 3.9
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+        
+    - uses: actions/checkout@v2
+    - name: Prepare environment
+      run: .github/workflows/prepare_environment.sh ${{ matrix.python_version }}
+        
+    - name: Run pylint
+      run: |
+        . "${CONDA}/bin/activate" test-env
+        list="$(find . -name \*.py -and -not -name '__init*' -and -not -name 'setup*')"
+        for file in $list; do pylint $file --max-statements=100; done
+        
+    - name: Run pytest
+      run: |
+        . "${CONDA}/bin/activate" test-env
+        pytest tests
+
+    - uses: actions/checkout@v2
+    - name: Artifacts for integration test
+      run: python src/db12/__main__.py > result_python${{ matrix.python-version }}.txt
```

I think there is a typo here, could you check?
```suggestion
      run: python src/db12/__main__.py > result_python${{ matrix.python_version }}.txt
```